### ISSUE TITLE
permissions page has very large rows

### ISSUE THREAD

**Issue by oalders** [metacpan/metacpan-web#2807]
Not sure what happened with the redesign, but I'm finding the permissions layout difficult to read.

There is the empty sidebar but also the exaggerated height of the "Authorized Releasers" rows.

<img width="1276" alt="Screenshot 2022-11-01 at 17 19 20" src="https://user-images.githubusercontent.com/96205/199343950-a99dbd8d-417e-4d79-b177-07ec66c5691a.png">

**Comment by dhogan8** [comment]
Thanks, I'll look into fixing the spacing issue.

As for the sidebar, is it supposed to be empty or should we be adding the `no-sidebar` class to this page?
This is the HTML markup for https://metacpan.org/dist/CPAN-Repository/permissions, which indicates there are no items to be shown.

```
<div class="nav-list-container slidepanel">
  <ul class="nav-list ">
  </ul>
</div>
```

**Comment by oalders** [comment]
I think having a sidebar might be confusing because this page does not correspond with any particular release of the distribution. So, no sidebar at all would be appropriate, I think.

**Comment by haarg** [comment]
I believe the layout issues can be fixed with a combination of #2814 and #2816.

I think it would be nice to add a sidebar to the page, as having no navigation elements isn't great. But that's something new we'd have to create. I'm fine with removing the sidebar for now.
